src/locales/de/main.json

```diff
@@ -1,24 +1,24 @@
 {
```

Comment:
Do we need these files even though we have the `i18n.ts` file? Unless there's a process keeping the two i18n sources in sync, it doesn't seem wise to have them in more than one place.